src/tools/rust-analyzer/crates/parser/src/syntax_kind/generated.rs

```diff
@@ -497,4 +497,3 @@ impl SyntaxKind {
-pub use T;
```

Comment:
This file is generated automatically, that line also needs to be removed from `src/tools/rust-analyzer/crates/syntax/src/tests/sourcegen_ast.rs`.